File: Famoser.MassPass.Data/Entities/Communications/Request/CollectionEntriesRequest.cs
using System;
using System.Collections.Generic;
using Famoser.MassPass.Data.Entities.Communications.Request.Base;

namespace Famoser.MassPass.Data.Entities.Communications.Request
{
    public class CollectionEntriesRequest : ApiRequest
    {
        public CollectionEntriesRequest()
        {
            KnownContentIds = new List<Guid>();
        }
        public Guid CollectionId { get; set; }
        public List<Guid> KnownContentIds { get; set; }
    }
}

File: Famoser.MassPass.Api/src/Models/Request/CollectionEntriesRequest.php
<?php
/**
 * Created by PhpStorm.
 * User: famoser
 * Date: 22/05/2016
 * Time: 23:01
 */

namespace Famoser\MassPass\Models\Request;


use Famoser\MassPass\Models\Request\Base\ApiRequest;

class CollectionEntriesRequest extends ApiRequest
{
    public $Guid;
    
    /*
     * @var string[]
     */
    public $KnownServerIds;
}